client/src/components/aggregations/utils.js

```diff
@@ -256,3 +256,32 @@ export function reportsToEvents(reports, showInterlocutors) {
         r1.title.localeCompare(r2.title)
     )
 }
+
+export function eventToCalendarEvents(events) {
+  return events
+    .map(event => {
+      // If no other data available title is the location name
+      const title = `${event.name}@${event.location.name}`
+      const start = new Date(event.startDate)
+      start.setSeconds(0, 0) // truncate at the minute part
+      const end = new Date(event.endDate)
+      end.setSeconds(0, 0) // truncate at the minute part
+      return {
+        title,
+        start,
+        end,
+        url: Event.pathFor(event),
+        extendedProps: { ...event },
+        allDay: true
```

Comment:
Shouldn't this be:
```javascript
        allDay: !Settings.engagementsIncludeTimeAndDuration
```
, since that setting is what you also use when showing the input field?
[BTW Maybe there should be a separate setting `eventsIncludeStartAndEndTime`?]